ansible/playbooks/molecule_ec2_teleop/molecule/teleop_server_check_desktop_icons_docker_ec2/molecule.yml

```diff
@@ -1,6 +1,7 @@
 ---
 dependency:
-  name: galaxy
+  name: shell
+  command: bash -c "ansible-galaxy collection install amazon.aws"
```

Comment:
use ansible modyle instead of shell here as well?

Reply:
No. Using depency: galaxy and specifying collections in requirements.yml doesn't actually install the collection, as I found out when I tried it. The only way to make sure the collection was actually installed was to use this shell and bash -c ansible galaxy collection install. Without it, you get errors saying: collection specified in requirements.yml, but not installed.